containers/flatpak/prepare

```diff
@@ -78,8 +58,14 @@ prepare() {
 
     TODAY="$(date +%F)"
 
-    subst metainfo.xml
-    subst yml
+    sed \
```

Comment:
I kinda liked the separate `subst()` function -- shortens the function a bit, and we might need it for other files, too. But bikeshedding, and very weak opinion.